src/apps/frontend/services/auth.service.ts

```diff
@@ -3,11 +3,25 @@ import { AccessToken, ApiResponse } from '../types';
 import APIService from './api.service';
 
 export default class AuthService extends APIService {
+  signup = async (
+    firstName: string,
+    lastName: string,
+    username: string,
+    password: string,
+  ): Promise<ApiResponse<void>> =>
+    this.apiClient.post('/account', {
+      first_name: firstName,
+      last_name: lastName,
+      username: username,
+      password: password,
+    });
+
   login = async (
     username: string,
     password: string,
-  ): Promise<ApiResponse<AccessToken>> => this.apiClient.post('/access-tokens', {
-    username,
-    password,
-  });
+  ): Promise<ApiResponse<AccessToken>> =>
+    this.apiClient.post('/access-tokens', {
```

Comment:
This is incorrect.

We need to probably create a Javascript object by parsing JSON object returned by api client.

Reply:
Done